app/assets/v2/js/pages/leaderboard.js

```diff
@@ -67,4 +67,33 @@ $(document).ready(function() {
       document.location.href = document.location.pathname + '?cadence=' + $('#cadence').val() + '&keyword=' + $('#tech-keyword').val() + '&product=' + $('#product').val();
     }
   });
+
+  const observerOptions = {
+    rootMargin: '0px 0px 200px 0px',
+    threshold: 0
+  };
+
+  function preloadImage(image) {
+    const src = image.getAttribute('data-src');
+
+    if (!src) {
+      return;
+    }
+    image.src = src;
+  }
+
+  let observer = new IntersectionObserver(function(entries, self) {
+    entries.forEach(entry => {
+      if (entry.isIntersecting) {
+        preloadImage(entry.target);
+        self.unobserve(entry.target);
+      }
+    });
+  }, observerOptions);
+
+  let images = document.querySelectorAll('[data-src]');
```

Comment:
Hey @mf2199 can you comment on why you decided to use observer here instead of just adding lazy to the images?